docs/markdown/Build-options.md

```diff
@@ -244,6 +244,9 @@ project which also has an option called `some_option`, then calling
 `yield` is `false`, `get_option` returns the value of the subproject's
 option.
 
+*Since 1.8.0*, it is possible to override the value of a yielding
```

Comment:
```suggestion
*Since 1.8.0*, it is possible to set or override the value of a yielding
```

Reply:
It was already possible to set it (to a non default value), but all subprojects had to use the same value. Do you mean "set" as "change the value while the top project keeps its default"?

Reply:
Yes, if you passed *only*  `-Dsubproj1:opt=value` it would be ignored. You had to set `-Dopt=value` and have it propagate everywhere. I guess we should rephrase it to make it clearer.